easybuild/easyconfigs/p/PyTables/pyTables-3.4.2-fix-libs.patch

```diff
@@ -0,0 +1,12 @@
+--- PyTables-3.4.2/setup.py.orig	2017-11-10 17:26:25.248733569 +0100
```

Comment:
@tovrstra This same patch is included in #5325, but with an extra comment on top, so different checksum.

Please steal it from there instead (and fix the checksum), to avoid conflicts if this gets merged first. ;-)